subprojects/shipkit/src/main/groovy/org/shipkit/gradle/configuration/ShipkitConfiguration.java

```diff
@@ -409,19 +423,20 @@ public void setContributors(Collection<String> contributors) {
             validateTeamMembers(contributors);
             store.put("team.contributors", contributors);
         }
-    }
 
-    /**
-     * Provides 'lenient' copy of this configuration instance,
-     * that does not fail fast when one accesses a property that is not configured (e.g. is null).
-     * <p>
-     * By default, release configuration object fails fast in this scenario.
-     * This is a good default because it helps us identify missing configuration early.
-     * However sometimes we want to check if the user has configured a property (like for GitHub write token) without failing.
-     *
-     * @return lenient copy of this configuration instance
-     */
-    public ShipkitConfiguration getLenient() {
-        return new ShipkitConfiguration(store.getLenient());
+        /**
+         * Contributors to be ignored in release notes and generated pom file.
+         * It should be VCS login
```

Comment:
It would be good to show an example of VCS login, let's make this comment compatible with the one for getContributors and also mention what happens if user is included in both contributors and ignoredContributors.